### DISCUSSION TITLE
Linux support / Ways to run FASTER on Linux

### DISCUSSION THREAD

**Discussion by byErikas** [Foxlider/FASTER#132] [Q&A] [answered]
Hey,

is there any way to run FASTER on Linux, and use it to manage the server instances? 
The tool is amazing itself, I love it, but I can't seem to be able to use it with Linux environments. Is this a knowledge issue, or is there no planned release/support for OSes other than Windows?

**Comment by Foxlider** [comment, marked as answer]
Short answer : no

Long answer : Nope. FASTER is built as a Windows App with the WPF framework. Linux support would mean remaking the entire UI with another framework like QT. 
Also, Arma is not build to run on Linux either. It is "supported" but it's more of a translated version of the Windows code

**Reply by byErikas** [reply]
Ah, I see. Thank you for a quick answer!
